sycl/doc/extensions/proposed/SYCL_EXT_ONEAPI_GROUP_STATIC_ARRAY.asciidoc

```diff
@@ -0,0 +1,590 @@
+= SYCL_EXT_ONEAPI_GROUP_STATIC_ARRAY
```

Comment:
If I'm being pedantic, I guess "array" isn't strictly accurate (because the new overloads accept a `std::span`).

I don't think the length of the name is an issue; the only time people will see the extension name is if they use the feature-test macro, and a feature test macro that people can understand without looking it up is a good thing imho.

Reply:
Makes sense. Users can also create an alias to hide the macro if it's too long for them. What do you think about `SYCL_EXT_ONEAPI_GROUP_ALGORITHM_SPAN_INTERFACE`?